sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/EditContactSteps.java

```diff
@@ -98,6 +100,133 @@ public EditContactSteps(WebDriverHelpers webDriverHelpers) {
+          softly.assertThat(aContact.getReportDate()).isEqualTo(editedContact.getReportDate());
+          softly
+              .assertThat(aContact.getReportingDistrict())
+              .isEqualToIgnoringCase(editedContact.getReportingDistrict());
+          softly
+              .assertThat(aContact.getResponsibleRegion())
+              .isEqualToIgnoringCase(editedContact.getResponsibleRegion());
+          softly
+              .assertThat(aContact.getResponsibleDistrict())
+              .isEqualToIgnoringCase(editedContact.getResponsibleDistrict());
+          softly
+              .assertThat(aContact.getResponsibleCommunity())
+              .isEqualToIgnoringCase(editedContact.getResponsibleCommunity());
+          softly
+              .assertThat(aContact.getReturningTraveler())
+              .isEqualToIgnoringCase(editedContact.getReturningTraveler());
+          softly
+              .assertThat(aContact.getCaseIdInExternalSystem())
+              .isEqualToIgnoringCase(editedContact.getCaseIdInExternalSystem());
+          softly
+              .assertThat(aContact.getCaseOrEventInformation())
+              .isEqualToIgnoringCase(editedContact.getCaseOrEventInformation());
+          softly
+              .assertThat(aContact.getIdentificationSource())
+              .isEqualToIgnoringCase(editedContact.getIdentificationSource());
+          softly
+              .assertThat(aContact.getIdentificationSourceDetails())
+              .isEqualToIgnoringCase(editedContact.getIdentificationSourceDetails());
+          softly
+              .assertThat(aContact.getTypeOfContact())
+              .isEqualToIgnoringCase(editedContact.getTypeOfContact());
+          softly
+              .assertThat(aContact.getAdditionalInformationOnContactType())
+              .isEqualToIgnoringCase(editedContact.getAdditionalInformationOnContactType());
+          softly
+              .assertThat(aContact.getContactCategory())
+              .isEqualToIgnoringCase(editedContact.getContactCategory());
+          softly
+              .assertThat(aContact.getRelationshipWithCase())
+              .isEqualToIgnoringCase(editedContact.getRelationshipWithCase());
+          softly
+              .assertThat(aContact.getDescriptionOfHowContactTookPlace())
+              .isEqualToIgnoringCase(editedContact.getDescriptionOfHowContactTookPlace());
+          softly
+              .assertThat(aContact.getProhibitionToWork())
+              .isEqualToIgnoringCase(editedContact.getProhibitionToWork());
+          softly
+              .assertThat(aContact.getHomeBasedQuarantinePossible())
+              .isEqualToIgnoringCase(editedContact.getHomeBasedQuarantinePossible());
+          softly
+              .assertThat(aContact.getQuarantine())
+              .isEqualToIgnoringCase(editedContact.getQuarantine());
+          softly.assertThat(aContact.getHighPriority()).isEqualToIgnoringCase("on");
```

Comment:
Please change `getHighPriority` into a boolean type because it is a checked or not checked option, and compare only objects not object vs value.

Reply:
reworked. added check as text for checbox's label for making sure we check the right one.